tests/utils/test_utils.py

```diff
@@ -82,3 +82,50 @@ class Foo:
         pass
 
     assert _get_fully_qualified_class_name(Foo()) == f"{__name__}.Foo"
+
+
+def test_inspect_original_var_name():
+    from mlflow.utils import _inspect_original_var_name
+    cap_name = None
+
+    def f1(a1):
+        nonlocal cap_name
+        cap_name = _inspect_original_var_name(a1, 'unknown')
+
+    xyz1 = object()
+    f1(xyz1)
+    assert cap_name == 'xyz1'
```

Comment:
Can we simplify the code here like below?

```suggestion
    def f1(a1, expected_var_name):
        assert _inspect_original_var_name(a1, 'unknown') == expected_var_name
        
    xyz1 = object()
    f1(xyz1)
```